Restart your shell session

Bonsitar opened the pull request:
according to: https://github.com/Linuxbrew/brew/issues/711

c0dechief committed:
one more step added to <EOS>.

according to: https://github.com/Linuxbrew/brew/issues/711

sjackman commented on install:
```diff
@@ -383,13 +383,13 @@ Dir.chdir HOMEBREW_REPOSITORY do
     curl_flags += "k" if mac? && macos_version < "10.6"
     core_tap = "#{HOMEBREW_PREFIX}/Homebrew/Library/Taps/homebrew/homebrew-core"
-    system "/bin/bash -o pipefail -c 'curl -#{curl_flags} #{BREW_REPO}/tarball/master | /bin/tar xz -m --strip 1'"
+    system "/bin/bash -o pipefail -c '/usr/bin/curl -#{curl_flags} #{BREW_REPO}/tarball/master | /bin/tar xz -m --strip 1'"
```
Please revert this unnecessary change.

sjackman commented on install:
```diff
@@ -383,13 +383,13 @@ Dir.chdir HOMEBREW_REPOSITORY do
     system "/bin/mkdir", "-p", core_tap
     Dir.chdir core_tap do
-      system "/bin/bash -o pipefail -c 'curl -#{curl_flags} #{CORE_TAP_REPO}/tarball/master | /bin/tar xz -m --strip 1'"
+      system "/bin/bash -o pipefail -c '/usr/bin/curl -#{curl_flags} #{CORE_TAP_REPO}/tarball/master | /bin/tar xz -m --strip 1'"
```
Ditto.

sjackman commented on install:
```diff
@@ -432,6 +432,7 @@ if !mac?
     PATH="#{HOMEBREW_PREFIX}/bin:$PATH"
 - We recommend that you install GCC by running:
     brew install gcc
+- After modifying your shell profile, you may need to restart session (logout & then log back in) if the brew command isn't found.
```
Change `restart session` to `restart your session` and `&` to `and`.

c0dechief committed:
adjusting pull request!

not sure why those mistakes and unnecessary changes happened but here I hope it changes in a right way!

ghost commented:
now I think it fixed! @sjackman

sjackman commented on install:
```diff
@@ -384,7 +384,7 @@ Dir.chdir HOMEBREW_REPOSITORY do
     core_tap = "#{HOMEBREW_PREFIX}/Homebrew/Library/Taps/homebrew/homebrew-core"
     system "/bin/bash -o pipefail -c 'curl -#{curl_flags} #{BREW_REPO}/tarball/master | /bin/tar xz -m --strip 1'"
-
+    
```
Almost there, @Bonsitar ! You've added some additional white space here at line 387. Please remove them.

maxim-belkin commented on install:
```diff
@@ -432,6 +432,7 @@ if !mac?
     PATH="#{HOMEBREW_PREFIX}/bin:$PATH"
 - We recommend that you install GCC by running:
     brew install gcc
+- After modifying your shell profile, you may need to restart your session (logout and then log back in) if the brew command isn't found.
```
I'd suggest breaking this line somewhere as it is too long.

sjackman replied:
Let's add a line break after `session`.
```
- After modifying your shell profile, you may need to restart your session
  (logout and then log back in) if the brew command isn't found.
```

c0dechief committed:
some other fixes...

ghost commented:
@maxim-belkin  @sjackman 
Now is this OK?

maxim-belkin commented on install:
```diff
@@ -432,6 +431,8 @@ if !mac?
 - We recommend that you install GCC by running:
     brew install gcc
+- After modifying your shell profile, you may need to restart your session
+(logout and then log back in) if the brew command isn't found.
```
I'd move `(logout` under `After`
```
- After modifying...
  (logout
```

sjackman commented on install:
```diff
@@ -384,7 +384,6 @@ Dir.chdir HOMEBREW_REPOSITORY do
     curl_flags += "k" if mac? && macos_version < "10.6"
     core_tap = "#{HOMEBREW_PREFIX}/Homebrew/Library/Taps/homebrew/homebrew-core"
     system "/bin/bash -o pipefail -c 'curl -#{curl_flags} #{BREW_REPO}/tarball/master | /bin/tar xz -m --strip 1'"
-
```
Please don't delete the blank line here. The empty blank line is desirable. Previously you also had spaces on this line.

Bonsitar replied:
@sjackman Oops! 
Why? As far as I know there is no deference since the command will skip that line! 
But I think it fixed now!

c0dechief committed:
hope to fixed correctly :(

c0dechief committed:
a little whitespace can show the lines more readable

maxim-belkin commented on install:
```diff
@@ -432,6 +432,8 @@ if !mac?
 - We recommend that you install GCC by running:
     brew install gcc
+- After modifying your shell profile, you may need to restart your session
+  (logout and then log back in) if the brew command isn't found.
```
brew -> `brew`

sjackman commented:
Merged. Thank you for your first contribution to Linuxbrew, Mohammad! 🏅

ghost commented:
Yaaaay!!! Thanks for your helps to merge this pull request in a right way!

sjackman commented:
Our pleasure!
